### ISSUE TITLE
libdart-core4.1 should conflict with libdart-core4.0

### ISSUE THREAD

**Issue by scpeters** [dartsim/dart#211]
I'm getting a failure when trying to install libdart-core4.1:
~~~
Unpacking libdart-core4.1 (4.1.0-ppa1~trusty) ...
dpkg: error processing archive /var/cache/apt/archives/libdart-core4.1_4.1.0-ppa1~trusty_amd64.deb (--unpack):
 trying to overwrite '/usr/lib/libdart-core4.so', which is also in package libdart-core4.0 4.0.0-ppa2~trusty
~~~
@j-rivero

We can probably make a similar change as the one in #202 / #203

**Comment by jslee02** [comment]
I believe this is fixed by #286 since `libdart-core4.x` installs `libdart-core4.so.*` (not `libdart-core4.so`).

**Comment by scpeters** [comment]
Fixed by #286
